frontend/src/metabase/query_builder/components/DataSelector/DataSelectorTablePicker/DataSelectorTablePicker.unit.spec.tsx

```diff
@@ -1,43 +1,67 @@
 import { checkNotNull } from "metabase/core/utils/types";
 import { getMetadata } from "metabase/selectors/metadata";
-import { SchemaId, TableId } from "metabase-types/api";
+import { InitialSyncStatus, Schema, Table } from "metabase-types/api";
 import { createMockDatabase, createMockTable } from "metabase-types/api/mocks";
 import { createMockState } from "metabase-types/store/mocks";
 import { createMockEntitiesState } from "__support__/store";
-import { renderWithProviders, screen } from "__support__/ui";
+import { renderWithProviders, screen, within } from "__support__/ui";
 import DataSelectorTablePicker from "./DataSelectorTablePicker";
 
 const TEST_TABLE = createMockTable();
 const TEST_DATABASE = createMockDatabase({ tables: [TEST_TABLE] });
+const NOT_SYNCED_DB_STATUSES: InitialSyncStatus[] = ["aborted", "incomplete"];
 
 interface SetupOpts {
-  schemaIds?: SchemaId[];
-  tableIds?: TableId[];
+  schemas?: Schema[];
+  tables?: Table[];
 }
 
-const setup = ({ schemaIds = [], tableIds = [] }: SetupOpts = {}) => {
+const setup = ({ schemas = [], tables = [] }: SetupOpts = {}) => {
```

Comment:
By looking at other tests it seems that this `setup` function needs to be fixed. It should only accept a `Database` object (from `metabase-types/api`) and pass all tables and schemas from the db to the underlying component. Something like this:

```
const setup = (opts: SetupOpts) => {
  const state = createMockState({
    entities: createMockEntitiesState({ databases: [opts.database]})
  });
 const metadata = getMetadata(state);
 const database = checkNotNull(metadata.database(opts.database.id));
 const schemas = database.getSchemas();
 const tables = database.getTables();

  renderWithProviders(
    <DataSelectorTablePicker
      selectedDatabase={database}
      schemas={metadataSchemas}
      tables={metadataTables}
      onChangeTable={jest.fn()}
    />,
    { storeInitialState: state },
  );
};
```

Could you fix the function similarly to this?

Reply:
Later in tests you would call 
```
setup({
  database: createMockDatabase({
    tables: createMockTable(...)
  }),
});
```

Reply:
Fixed, thanks